zerver/models/realms.py

```diff
@@ -759,6 +764,15 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
             default_group_name=SystemGroups.EVERYONE,
             id_field_name="can_delete_own_message_group_id",
         ),
+        can_invite_users_group=GroupPermissionSetting(
+            require_system_group=False,
+            allow_internet_group=False,
+            allow_owners_group=False,
```

Comment:
Should `allow_owners_group=True` here?

Reply:
Yeah. We might want to do a broader audit; I think now that we support anything, it's possible that `allow_owners_group=False` is not a feature we need at all, and we could just remove it; I think the settings where such a group doesn't make sense already have a fixed list of options (like just everyone or "members").

Reply:
I opened https://github.com/zulip/zulip/issues/32401 to track this.